pkgs/by-name/ne/netfetch/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+}:
+let
+  version = "5.2.10";
+in
+buildGoModule {
+  pname = "netfetch";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "deggja";
+    repo = "netfetch";
+    rev = "${version}";
+    hash = "sha256-N3wKpWdG92cXH0TwAkcsld9TRrfPRkbw0uZY/X4d+xk=";
+  };
+
+  vendorHash = "sha256-/Em3hx5tiQjThLBPJDHGsqxUV3eXeymJ5pY9c601OW0=";
+  proxyVendor = true;
```

Comment:
why was this needed here?

Reply:
I encountered this issue: https://github.com/NixOS/nixpkgs/issues/203039
Due to the main project making use of go workplaces.